less_slow.cpp

```diff
@@ -3326,8 +3393,13 @@ void parse_regex(bm::State &state, std::string_view config_text) {
     std::size_t pairs = 0, bytes = 0;
     std::vector<std::pair<std::string, std::string>> settings;
 
+#if defined(_MSC_VER)
+    // MSVC does not implement std::regex_constants::multiline yet
+    auto regex_options = std::regex_constants::ECMAScript;
+#else
     // Use multiline mode so ^ and $ anchor to line breaks.
     auto regex_options = std::regex_constants::ECMAScript | std::regex_constants::multiline;
+#endif
```

Comment:
Maybe replace this with a simple `if` or ternary condition as opposed to `#if` macro?

Reply:
I'm not sure how that would work at runtime, because MSVC does not define std::regex_constants::multiline at all, so any mention of it would not compile.